Testscripts/Linux/ETHTOOL-CHECK-CHANNEL.sh

```diff
@@ -35,11 +35,17 @@ if ! GetSynthNetInterfaces; then
     exit 0
 fi
 
+if [[ $DISTRO != "ubuntu_x"* ]]; then
```

Comment:
why strict on ubuntu

Reply:
12.04 and 14.04 do not support.

Reply:
other distro like centos, redhat, suse also don't support?

Reply:
Oops I merged it. Possibly but this is not my scope for today. Will check later for those legacy ones.

Reply:
@LiliDeng  I found other distro like coreos and centos like this case. I have a big matrix for those work. I will open new PRs soon.